hack/verify-release.py

```
@@ -0,0 +1,988 @@
+
+
+def _get(url, *, retries=3, delay=2, **kwargs) -> requests.Response:
+    """
+    Wrapper around requests.get with retry logic for transient network errors
+    (DNS resolution failures, connection resets, timeouts).  Raises
+    requests.RequestException with a concise message on final failure.
+    """
+    last_exc = None
+    for attempt in range(retries):
+        try:
+            return requests.get(url, **kwargs)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)